common/djangoapps/util/query.py

```diff
@@ -7,3 +8,15 @@ def use_read_replica_if_available(queryset):
     If there is a database called 'read_replica', use that database for the queryset.
     """
     return queryset.using("read_replica") if "read_replica" in settings.DATABASES else queryset
+
+
+def get_read_replica_cursor_if_available(db):  # pylint: disable=invalid-name
+    """
+    Returns cursor to read_replica or default if not available
+    """
+    try:
+        cursor = db.connections['read_replica'].cursor()
```

Comment:
ah, you are looking for the read_replica, great! That was my main concern, and whether or not this conflicted with getting ORA2 data into the data package from Analytics @shnayder 

I think for production, we'd want to force the use of the read replica, and never let this use the live DB.